siliconcompiler/tools/yosys/syn_asic.tcl

```diff
@@ -163,6 +163,16 @@ foreach bb_file $sc_blackboxes {
 # Synthesis
 ########################################################
 
+# Before working on the design, we mask out any module supplied via
+# `blackbox_modules`. This allows synthesis of parts of the design without having
+# to modify the input RTL.
+if { [dict exists $sc_cfg tool $sc_tool task $sc_task var blackbox_modules] } {
+    foreach bb [dict get $sc_cfg tool $sc_tool task $sc_task var blackbox_modules] {
+        puts "Blackboxing module: $bb"
```

Comment:
please use `yosys log ` to log the message for some reason `puts` doesn't get handled correctly in yosys. (also I noticed puts was used just above this, if you have the time to fix that too that would be great, if not don't worry about it)